CHANGELOG.md

```diff
@@ -5,8 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.6.2] - January 2nd 2020
```

Comment:
can we put this _below_ the Unreleased section?

Reply:
Sure